proposals/p2922.md

```diff
@@ -0,0 +1,153 @@
+-   [Alternatives considered](#alternatives-considered)
+    -   [Make no changes](#make-no-changes)
+    -   [Have some overlap between Carbon leads and the Conduct team](#have-some-overlap-between-carbon-leads-and-the-conduct-team)
+
+<!-- tocstop -->
+
+## Abstract
+
+The Carbon Project relied on the three leads to handle conduct concerns
+initially as it bootstrapped its community team and expertise. We now have an
+active and effective community lead and team of moderators. Our community lead
+has worked to train and ramp up a new and independent conduct team.
+
+This proposal both provides an overview of the process and hands off conduct
+handling to the new team! Going forward, we expect routine updates to the
+conduct team to happen without full proposals as they allow trained folks to
+rotate in and out of this difficult but essential role on the project.
+
+## Problem
+
+-   Our Code of Conduct team is made up of those with the highest power in the
+    community (Carbon leads), and our current framework does not allow for clear
+    accountability in terms of their own conduct due to this lack of separation.
+-   The diversity of backgrounds and perspectives is limited among our current
+    CoC team.
+-   The Carbon leads have limited time and availability to intervene in urgent
+    and important misconduct situations.
+
+So, it is time to build a specialized team to deal with escalations which is not
+subordinate to the Carbon leads, and is able to handle diverse misconduct
+escalations in an adequate and timely manner. This means we can recruit and/or
+train people from diverse backgrounds to handle such escalations for the Carbon
+community, and provide competent support to the leads, the rest of the
+moderation team, and everyone else in the community.
+
+TODO: What problem are you trying to solve? How important is that problem? Who
+is impacted by it?
+
+## Background
+
+Our Code of Conduct:
+https://github.com/carbon-language/carbon-lang/blob/trunk/CODE_OF_CONDUCT.md
+
+Our current CoC team:
+
+-   Chandler Carruth (@chandlerc on Discord and GitHub)
+-   Kate Gregory (@KateGregory on Discord and GitHub)
+-   Richard Smith (@zygoloid on Discord and GitHub)
+
+## Proposal
+
+Recruit and train 5+ potential new conduct team members to deal with
+conduct-related escalations on behalf of and for the Carbon community, and
+replace the current CoC team as soon as enough members are ready.
+
+We should always have at least 3 conduct team members available to respond
+promptly, even factoring in vacations, getting sick, or other normal
+disruptions.
+
+Subsequent updates to the exact members will be managed by the new conduct team
+themselves, and without full proposals but just PRs to update the relevant
+documentation.
+
+## Details
+
+### Initial team members
+
+-   TBA
+
+We will continue to look for more candidates in order to get to a point where we
+have a sustainable number, especially as the community grows.
+
+### Process
```

Comment:
Some parts of this section seem like they're intended to remain true over time (e.g. "New members will have access..." and "We will make sure...") rather than being tied to this proposal -- should those go in `conduct_team.md`?

Reply:
See above -- basically not sure this needs to happen in the critical path, and not sure I'm the best person to create the long-term documentation there.